tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -130,6 +130,13 @@ public void Setup_can_setup_generic_method()
 			Assert.Equal(3, handledMessages.Count);
 		}
 
+		[Fact]
+		public void Setup_can_automock()
+		{
+			this.protectedMock.Setup(m => m.Nested.Method(1)).Returns(123);
+			Assert.Equal(123, mock.Object.GetNested().Method(1));
+		}
+
```

Comment:
&dagger;) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.